packages/nextjs/app/page.tsx

```diff
@@ -1,67 +1,150 @@
 "use client";
 
+import { useState } from "react";
+import Image from "next/image";
 import Link from "next/link";
-import type { NextPage } from "next";
+import TicTacToeGame from "./TicTacToeGame";
+import { images, useGameState } from "./gameContext";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
 import { useScaffoldReadContract } from "~~/hooks/scaffold-eth";
 
-const Home: NextPage = () => {
+// Interface for RenderImagesInPyramid props
+interface RenderImagesInPyramidProps {
+  pyramidLevels: string[][];
+  showTicTacToe: boolean;
+  setShowTicTacToe: (value: boolean) => void;
+}
+
+// DetailsComponent function
+function DetailsComponent(): JSX.Element {
+  const { gameWon, setGameWon } = useGameState();
+  const [showTicTacToe, setShowTicTacToe] = useState<boolean>(false);
+
+  // Build pyramid levels
+  const pyramidLevels: string[][] = [];
+  const remainingImages: string[] = [...images];
+  let level: number = 1;
+  while (remainingImages.length > 0) {
+    const levelImages: string[] = remainingImages.splice(0, level);
+    pyramidLevels.push(levelImages);
+    level++;
+  }
+
+  return (
+    <div className="flex items-center flex-col flex-grow pt-10 bg-base-200 min-h-screen">
+      <h1 className="text-center text-5xl font-bold mb-12 text-[#D4A5A5] tracking-wide">Welcome to Batch 13</h1>
+
+      {showTicTacToe && !gameWon && (
```

Comment:
This control can be inside TicTacToeGame, if you don't want to show anything you can simply `return null`.